openreview/conference/templates/reviewRebuttalProcess.js

```diff
@@ -0,0 +1,86 @@
+        message: 'We have received your rebuttal on your submission to ' + SHORT_PHRASE + '.\n\nPaper number: ' + forum.number + '\n\nPaper title: ' + forum.content.title + '\n\n' + content
+      };
+      promises.push(or3client.or3request( or3client.mailUrl, authors_writer_mail, 'POST', token ));
+
+      if (USE_AREA_CHAIRS && (note.readers.includes('everyone') || note.readers.includes(AREA_CHAIRS_ID))) {
+        var areachair_mail = {
+          groups: [AREA_CHAIR_1_ID],
+          ignoreGroups: ignoreGroups,
+          subject : '[' + SHORT_PHRASE + '] Rebuttal posted to your assigned Paper number: ' + forum.number + ', Paper title: "' + forum.content.title + '"',
+          message: 'A submission to ' + SHORT_PHRASE + ', for which you are an official area chair, has received a rebuttal. \n\nPaper number: ' + forum.number + '\n\nPaper title: ' + forum.content.title + '\n\n' + content
+        };
+        promises.push(or3client.or3request( or3client.mailUrl, areachair_mail, 'POST', token ));
+      }
+
+      var reviewers_submitted = REVIEWERS_ID + '/Submitted';
+      if (note.readers.includes('everyone') || note.readers.includes(REVIEWERS_ID)) {
+        var reviewer_mail = {
+          groups: [REVIEWERS_ID],
+          ignoreGroups: review.signatures,
+          subject: '[' + SHORT_PHRASE + '] Rebuttal posted to your assigned Paper number: ' + forum.number + ', Paper title: "' + forum.content.title + '"',
+          message: 'A submission to ' + SHORT_PHRASE + ', for which you are a reviewer, has received a rebuttal. \n\nPaper number: ' + forum.number + '\n\nPaper title: ' + forum.content.title + '\n\n' + content
+        };
+        promises.push(or3client.or3request( or3client.mailUrl, reviewer_mail, 'POST', token ));
+      } else if (note.readers.includes(reviewers_submitted)) {
+        var reviewer_mail = {
+          groups: [reviewers_submitted],
+          ignoreGroups: review.signatures,
+          subject: '[' + SHORT_PHRASE + '] Rebuttal posted to your assigned Paper number: ' + forum.number + ', Paper title: "' + forum.content.title + '"',
+          message: 'A submission to ' + SHORT_PHRASE + ', for which you are a reviewer, has received a rebuttal. \n\nPaper number: ' + forum.number + '\n\nPaper title: ' + forum.content.title + '\n\n' + content
+        };
+        promises.push(or3client.or3request( or3client.mailUrl, reviewer_mail, 'POST', token ));
+      }
+
+      var review_signature_mail = {
+        groups: review.signatures,
+        subject : '[' + SHORT_PHRASE + '] Rebuttal posted to your review submitted - Paper number: ' + forum.number + ', Paper title: "' + forum.content.title + '"',
+        message: 'A submission to ' + SHORT_PHRASE + ', for which you are a reviewer, has received a rebuttal. \n\nPaper number: ' + forum.number + '\n\nPaper title: ' + forum.content.title + '\n\n' + content
```

Comment:
'A submission to ' + SHORT_PHRASE + ' that you reviewed has received a new rebuttal.'